packages/apps/plugins/plugin-help/src/components/Shortcuts/ShortcutsList.tsx

```diff
@@ -16,7 +15,7 @@ export const ShortcutsList = () => {
   const bindings = Keyboard.singleton.getBindings();
 
   // TODO(burdon): Factor out.
-  const toString = (label: Label) => (Array.isArray(label) ? t(...label) : label);
+  const toString = (label: any) => (Array.isArray(label) ? t(label[0], label[1]) : label);
```

Comment:
These function(s) (should be one somewhere, tricky because it depends on the `TFunction` type) should have a different name than the built-in `toString` method.

`toLocalizedString` perhaps?

Reply:
thanks for the callout, I had meant to circle back to this, this was primarily to get the build passing lol